google/cloud/forseti/notifier/notifiers/slack_webhook.py

```diff
@@ -69,16 +70,19 @@ def _compose(self, violation):
         return ('*type*:\t`{}`\n*details*:\n'.format(self.resource) +
                 self._dump_slack_output(violation.get('violation_data'), 1))
 
+    # Wait 30 seconds before retrying: https://api.slack.com/docs/rate-limits
+    @retry(wait_exponential_multiplier=30000, wait_exponential_max=60000,
```

Comment:
We should specify the exception to retry instead of retrying on all general exceptions.

Reply:
Updated to retry when retryable exceptions only; 429 response status code included.